Kernel/Library/StdLib.h

```diff
@@ -16,6 +17,25 @@
 #include <stddef.h>
 
 ErrorOr<NonnullOwnPtr<Kernel::KString>> try_copy_kstring_from_user(Userspace<char const*>, size_t);
+
+ErrorOr<void> try_copy_string_from_user(Userspace<char const*>, char*, size_t);
+
+template<size_t Size>
+ErrorOr<void> try_copy_string_from_user_into_static_char_buffer(Userspace<char const*> user_str, FixedCharBuffer<Size>& buffer, size_t user_str_size)
```

Comment:
This name is misleading. There is no requirement that the FixedCharBuffer have static storage duration.